pkg/platform/kube/ingress/ingress.go

```diff
@@ -367,7 +386,7 @@ func (im *Manager) DeleteIngressByName(ingressName string, namespace string,  de
 				return errors.Wrap(err, "Failed to get ingress resource on ingress deletion by name")
 			}
 
-			im.logger.DebugWith("Ingress resource not found. Aborting deletion",
+			m.logger.DebugWith("Ingress resource not found. Aborting deletion",
```

Comment:
imo, change the logic to be postive in a way of
`if IsNotFound error { return nil} else { return wrappedError }`

what do you think?

Reply:
sure